classes/opponentPhaseDucks.lua

```diff
@@ -0,0 +1,83 @@
+-- D U C K S   B A Y B E
+
+local opponentPhaseDucks = {}
+opponentPhaseDucks.__index = opponentPhaseDucks
+
+opponentPhaseDucks.DEFAULT_BULLET_COOLDOWN = 0.4
+opponentPhaseDucks.DEFAULT_BULLET_SPEED = 500
+
+
+local NUM_DUCKS = 40
+local MAX_TURN_SPEED = 2
+
+--------------------
+-- MAIN CALLBACKS --
+--------------------
+
+function opponentPhaseDucks.new()
+	local self = classes.opponentBase.new(opponentPhaseDucks)
+
+	self.ducks = {}
```

Comment:
The ducks are kept track of by the `opponentPhaseDucks` object, rather than creating a new table `ducks` in the game state only for this phase